editor/inspector/contributions/node.js

```diff
@@ -1816,3 +1833,43 @@ exports.beforeClose = async function beforeClose() {
 exports.config = {
     section: require('../components.js'),
 };
+
+
+/**
+ * @description 比较节点树，并渲染节点树
+ * @param oldTree 旧树
+ * @param newTree 新树
+ * @param resultTree 结果树
+ *
+ * */
+function diffRenderTree(oldTree, newTree, resultTree) {
```

Comment:
Why do we need DIFF? In most cases, there should be no difference

Reply:
in new PR, I'm  not use DIFF, I will delete this function.